persistent/Database/Persist/Quasi.hs

```diff
@@ -216,7 +216,7 @@ Person sql=peoples
     name Text
 @
 
-== Change table/collection key definition (field name and/or type, persistent >= 2.1)
+== Change table/collection key definition (field name and\/or type, persistent >= 2.1)
```

Comment:
Escaping of the 2nd `/` was needed to render properly, see [this page](Change tablecollection key definition (field name andor type, persistent >= 2.1).